src/ensembl/src/content/app/help/Help.tsx

```diff
@@ -63,26 +71,31 @@ const AppBar = () => {
 
 const MainContent = (props: { article: ArticleData }) => {
   const { article } = props;
-  let content;
-  if (article.type === 'index') {
-    content = <IndexArticle article={article} />;
-  } else if (['article', 'video'].includes(article.type)) {
-    const renderedArticle =
-      article.type === 'article' ? (
-        <TextArticle article={article} />
-      ) : (
-        <VideoArticle video={article} />
-      );
-    content = (
-      <HelpArticleGrid className={styles.articleGrid}>
-        {renderedArticle}
-        {!!article.related_articles.length && (
-          <RelatedArticles articles={article.related_articles} />
-        )}
-      </HelpArticleGrid>
-    );
+  if (article.type !== 'article' && article.type !== 'video') {
+    return null;
   }
+  const renderedArticle =
+    article.type === 'article' ? (
+      <TextArticle article={article} />
+    ) : (
+      <VideoArticle video={article} />
+    );
+
+  const content = (
+    <HelpArticleGrid className={styles.articleGrid}>
+      {renderedArticle}
+      {!!article.related_articles.length && (
+        <RelatedArticles articles={article.related_articles} />
+      )}
+    </HelpArticleGrid>
+  );
+
   return <main className={styles.main}>{content}</main>;
 };
 
+const isIndexRoute = (pathname: string) => {
+  // handle both /help and /help/
+  return pathname.split('/').filter(Boolean).join() === 'help';
```

Comment:
Why not just `pathname.split('/')[1]`?

Reply:
I think `pathname.replace('/', '')` might be even better

Reply:
Updated.

Reply:
I think we need `replaceAll` to handle both `/help` & `/help/`

Reply:
👍 ✅